crates/compiler/gen_wasm/src/low_level.rs

```diff
@@ -541,12 +541,6 @@ impl<'a> LowLevelCall<'a> {
                 let elem_layout = backend.layout_interner.get(elem_layout);
                 let (elem_width, elem_align) =
                     elem_layout.stack_size_and_alignment(backend.layout_interner, TARGET_INFO);
-                let (spare_local, spare_offset, _) = ensure_symbol_is_in_memory(
-                    backend,
-                    spare,
-                    Layout::usize(TARGET_INFO),
-                    backend.env.arena,
-                );
```

Comment:
@brian-carroll here the `spare` value (how much spare capacity to create) is passed by-reference. The zig code expects it by-value. This is probably an incorrect copy-paste at some point. My fix works, but I would appreciate a quick look from you here.

Reply:
Gosh, sorry, didn't mean to leave it 2 days. I was doing non-programming stuff all weekend, didn't really look at GH!
Yeah, the old code looks wrong and weird, and your fix looks right! Bizarre that I passed this by reference, it doesn't make any sense. Copy-paste error is a good theory!